enterprise-suite/.ws.conf

```diff
@@ -0,0 +1,10 @@
+productName=Console Docker Images
+projectName=not applicable
+
+docker.scanImages=true
+docker.includes=.*enterprise-suite.* alpine busybox prom/.* .*kube-state-metrics janeczku/go-dnsmasq jimmidyson/configmap-reload
```

Comment:
Does this need versions?

Also - is it documented how this works somewhere? I'm not sure how this picks up the actual image names or knows the repo.

Reply:
docs are https://whitesource.atlassian.net/wiki/spaces/WD/pages/33718339/Unified+Agent

and

https://github.com/whitesource/fs-agent

Reply:
it does `docker images` against minikube and unpacks whatever matches the regex